index.html

```diff
@@ -319,47 +320,6 @@ <h2 class="title_contact">Contact me</h2>
 			</div>
 		</section>
```

Comment:
- [ ] The button should have interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the figma template. Kindly make sure your design for the interactions matches original design guidelines(colors, margins, padding etc)

![image](https://user-images.githubusercontent.com/43995830/154142030-6b7a832d-52bf-4f58-a9be-cbc6777e9493.png)

Reply:
![Screen Shot 2022-02-15 at 22 29 28](https://user-images.githubusercontent.com/88894525/154143965-028f9a9f-e5d5-488f-8385-1da062f5fdfb.png)

Is this one i put hover on all my buttons